main/getting-started/before-using-agoric.md

```diff
@@ -15,42 +16,53 @@ git clone https://github.com/Agoric/agoric-sdk
 cd agoric-sdk
 yarn install
 yarn build
-yarn link-cli ~/bin/agoric
-agoric --version
 ```
 
-Then proceed to [starting a project](/getting-started/start-a-project.md).
+### Add the `agoric` command to your PATH
```

Comment:
Adding a whole section here is somewhat awkward.

How about we just add:

```
export PATH=~/bin:$PATH
```

before `agoric --version` in this section, and then move the whole dedicated section on PATH to the expanded directions below?

Reply:
sounds good! thanks!